.github/workflows/licensed.yml

```diff
@@ -7,18 +7,9 @@ on:
   pull_request:
     branches:
       - main
+  workflow_dispatch:
```

Comment:
Why we are adding a dispatch event? I saw in the attached setup node PR as well

Reply:
@tiwarishub The `workflow_dispatch` event allows us to manually trigger a workflow and test it before creating a PR. I added by referring to `actions/setup-node`.